action.yml

```diff
@@ -29,8 +29,8 @@ inputs:
     bearerToken:
         description: 'Bearer Authentication Token'
         required: false
-    customHeaders:
```

Comment:
Hey I think `customHeaders` here shouldn't be overwritten? It still works, it's just that Github Actions would give you a warning that it's not of of the valid inputs

![image](https://user-images.githubusercontent.com/12206156/89253400-c5095780-d646-11ea-943d-f4df6780f416.png)

Reply:
Your Right. I will Fix ist asap

Reply:
Created  v1.41, thanks for Support